src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -1,4 +1,4 @@
-## Public HMDA - Data Fields with Values and Definitions
+# Public HMDA - Data Fields with Values and Definitions
```

Comment:
Data table for `agency_code` not displaying as a table
 
http://localhost:3000/documentation/2017/lar-data-fields/#agency_code

Reply:
Add a blank line between Values: and the table
```
- **Values:**

  |Value|Abbreviation|Name|
  |--|--|--|
  |**1**|_OCC_|Office of the Comptroller of the Currency|
  |**2**|_FRB_|Federal Reserve System|
  |**3**|_FDIC_|Federal Deposit Insurance Corporation|
  |**5**|_NCUA_|National Credit Union Administration|
  |**7**|_HUD_|Department of Housing and Urban Development|
  |**9**|_CFPB_|Consumer Financial Protection Bureau|
  ```

Reply:
Sweet! This has been updated.